index.html

```diff
@@ -543,6 +569,14 @@ <h2>Filter <var>buffer</var> by <var>name</var> and <var>type</var></h2>
         </li>
       </ol>
     </section>
+    <section data-link-for="PerformanceObserver">
+      <h2>Register performance entry type</h2>
+      <p>To register a performance entry type, run the following steps:</p>
```

Comment:
you should put "register a performance entry type" in a `<dfn>`, so that other specs can hook into it

Reply:
Once you add that, you should also link directly to the algorithm in line 447, rather than to the section

Reply:
Done